napari/utils/colormaps/colormap_utils.py

```diff
@@ -150,6 +150,22 @@
     }
 )
 
+DISCONTINUOUS_COLORMAPS = {
+    'HiLo': Colormap(
+        name='HiLo',
+        display_name='HiLo',
+        colors=[[0.0, 0.0, 0.0, 1.0], [1.0, 1.0, 1.0, 1.0]],
+        high_color=[1.0, 0.0, 0.0, 1.0],
+        low_color=[0.0, 0.0, 1.0, 1.0],
+    ),
+    'bad': Colormap(
+        name='bad',
+        display_name='bad',
+        colors=[[0.0, 0.0, 0.0, 1.0], [1.0, 1.0, 1.0, 1.0]],
+        bad_color=[1.0, 0.0, 0.0, 1.0],
+    ),
```

Comment:
LOL is this a common name for this colormap? Or do we want to call it nan also?

Reply:
lol nope, I just made it up, assuming I would get lots of comments about it :P Yeah, `nan` is also an option. I can imagine even more verbose, like `highlight_nan`

Reply:
going for simply `nan` for now.